_includes/proposals.html

```diff
@@ -33,11 +31,9 @@ <h4 class="featurelist__item__title flex-grow">
                   </li>
                 {% endfor %}
                 {% endif %}
-                {% if proposal.specification %}
-                <li class="featurelist__item__spec">
-                  <a href="{{ proposal.specification }}">Specification</a>
-                </li>
-                {% endif %}
+              <li class="featurelist__item__spec">
+                <a href="https://tc39.github.io/{{ proposal.specification }}">Specification</a>
```

Comment:
made a mistake during the rebase

```suggestion
                <a href="https://tc39.github.io/{{ proposal.id }}">Specification</a>
```